src/components/RouteOption/DockedEbikeRouteOption.jsx

```diff
@@ -105,7 +114,7 @@ DockingStationSelector.propTypes = {
       id: PropTypes.string.isRequired,
       name: PropTypes.string.isRequired,
     }),
-  ),
+  ).isRequired,
```

Comment:
good catch! thanks. if this was null, `stations.map` would crash. good to mark it as required